_includes/footer.html

```diff
@@ -44,6 +44,8 @@
         <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
       </div>
     </div>
+    <p>This site is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
+    </p>
```

Comment:
再利用する人がBYを満たすために原著作者を入れるとき、すぐに原著作者情報を見つけられるようにこのすぐ近くに原著作者情報も入れるのはどうですか？

```suggestion
    </p>
    <p>Copyright 2015-{{ site.time | date: "%Y" }} OSS Gate</p>
```

Reply:
確かに、自分もそのほうがいいと思いました！